Plugin metadata analyzer (#1595)
wix/wix-ui: packages/wix-storybook-utils/README.md

        test: /\.story\.js$/,
        loader: 'wix-storybook-utils/loader',
        options: {

>           plugins: {},

Reviewer: ```suggestion
          plugins: [],
```

>           metadataPlugins: {},

Reviewer: ```suggestion
          plugins: {},
```

  });
```

> _metadataPlugins_ object above is optional but it can be used to add custom parsers.

Reviewer: ```suggestion
optional _plugins_ array can be used to change the output of default parser.
```
